### PR TITLE
Migrate accessibility tests to Cypress and GitHub Actions

### PR DESCRIPTION
**Pull request by ddzz**
## Description
This PR migrates the daily accessibility scan to Cypress. Tests will run on a daily schedule with Slack notifications. The existing Nightwatch tests will be removed in a separate PR.

## Acceptance criteria
- [ ] Cypress a11y tests run and pass on GitHub Actions
- [ ] #-daily-accessibility-scan Slack channel is notified of results of test runs

### CODE REVIEW

**Comment by timwright12** [comment]
converted to draft since it seems like you're still working on this

**Comment by ddzz** [comment]
> converted to draft since it seems like you're still working on this

Thanks, it's almost ready 🤞

**Review by 1Copenut — APPROVED** [review]
I'm 💯 for these changes. We've worked closely on making sure things are running the way we expect. I'll notify the VA 508 office the static content checks don't include the async map load and CC our PMs.

**Review by timwright12 — APPROVED** [review]

File: .github/workflows/a11y.yml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+      - name: Get Slack bot token
+        uses: marvinpinto/action-inject-ssm-secrets@v1.1.1
+        with:
+          ssm_parameter: /devops/github_actions_slack_bot_user_token
+          env_variable_name: SLACK_BOT_TOKEN
+
+      - name: Accessibility Test Failure
+        uses: ./.github/actions/vsp-github-actions/slack-socket
+        if: ${{ failure() }}
```
Comment (CBonade) [inline .github/workflows/a11y.yml:231]: in `vets-website` in order to reference success/failure in a prior job, we had to reference it as `${{needs.a11y.result == 'failure'}}`.  Are we sure this works in this syntax?

**Review by CBonade — APPROVED** [review]
Just one question about the failure/success states on the slack step. If all is confirmed, LGTM!

File: .github/workflows/a11y.yml (hunk 1 above)
Comment (ddzz) [reply .github/workflows/a11y.yml:231]: Good catch, I'll fix this.

**Pull request merged**